### ISSUE TITLE
neural network visualisation with netron (slightly useful)

### ISSUE THREAD

**Issue by tampueroc** [lucasmurray97/deep-crowns#1]
unrelated e.g
![image](https://github.com/lucasmurray97/deep-crowns/assets/46233260/d5139cfd-1f32-4a3c-8134-015d1c77c142)

compatible PyTorch export for Netron
`torch.onnx.export(model, X_test, 'iris.onnx', input_names=["features"], output_names=["logits"])`

references
https://github.com/lutzroeder/netron
https://github.com/christianversloot/machine-learning-articles/blob/main/visualizing-your-neural-network-with-netron.md
https://machinelearningmastery.com/visualizing-a-pytorch-model/
